libraries/AP_HAL_Linux/AP_HAL_Linux_Namespace.h

```diff
@@ -30,6 +30,7 @@ namespace Linux {
     class LinuxRCOutput_AioPRU;
     class LinuxRCOutput_Navio;
     class LinuxRCOutput_ZYNQ;
+    class LinuxRCOutput_Bebop;
```

Comment:
It is exclusive to the Bebop currently. We will probably add new supported boards. I would prefer to leave it this way now untill I have a clearer view of what's coming and then rename it if needed. I will change this name at the moment there is a new supported board. Do you see a problem with that?